kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/UserTaskServicesClient.java

```diff
@@ -110,6 +110,8 @@
 
     TaskInstance findTaskById(Long taskId);
 
+    TaskInstance findTaskByIdWithSLA(Long taskId);
```

Comment:
I still think that in terms of keeping a standard compare to the same API we should have two methods: findTaskById(Long taskId) and findTaskById(Long taskId, Boolean withSla), that would mean just that the default would be false for reusing the new method, keeping the expected previous behaviour. See an example: https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/UserTaskServicesClient.java#L102
Usually we also only include what is been filtered on on the method name. My 2cents

Reply:
@cristianonicolai for me this makes sense. That was something that we talk throw the PR comments. @MarianMacik are we good with this change?

Reply:
@cristianonicolai @nmirasch OK, you are right. I thought that we should avoid doing that, but on the other hand we already have this standard. The improvement I suggested would be just a drop in the ocean :) I agree it's better to be consistent.